packages/builder/builder-shared/src/types/bundlerConfig.ts

```diff
@@ -22,6 +22,45 @@ type SplitChunks = Configuration extends {
   : never;
 
 type WebpackOptimization = NonNullable<Configuration['optimization']>;
+type WebpackResolve = NonNullable<Configuration['resolve']>;
+type WebpackOutput = NonNullable<Configuration['output']>;
+
+// fork from the @rspack/core
```

Comment:
Why fork this type

Reply:
1. We need the Rspack type in `bulider-shared` to define some config or type
2. the `@rspack/core`  bundler size is so big that we can't  integration into `builder-shared`

so I fork the type temporarily